akka-projection-grpc/src/main/scala/akka/projection/grpc/producer/javadsl/Transformation.scala

```diff
@@ -45,6 +53,14 @@ final class Transformation private (private[akka] val delegate: scaladsl.EventPr
     new Transformation(delegate.registerMapper[A, B](event => f.apply(event).asScala))
   }
 
+  def registerLowLevelMapper[A, B](
```

Comment:
instead of "LowLevel" I would prefer to name it `registerAsyncEnvelopeMapper`

and I think it should include Async to align with the naming of the others returning CompletionStage